src/main/app/src/components/Planning.js

```diff
@@ -47,6 +49,32 @@ const LoadedView = (classes) => {
   const activityTableRef = useRef();
   //className={classes.invisible}
 
+  const [openAlert, setOpenAlert] = React.useState(false);
+  const [alertText, setAlertText] = React.useState();
+
+  const AlertSnackbar = () => {
```

Comment:
Here is the other duplicate `AlertSnackbar`. The component is identical to the `AlertSnackbar` in `NewPurchaseOrderForm.js` on line 102